src/strings.ts

```diff
@@ -0,0 +1,7 @@
+export function compareStrings(a: string, b: string) {
+  return a.localeCompare(b, "en", { sensitivity: "base" });
+}
+
+export function stripPackageSearchPunctuation(name: string) {
+  return name.toLowerCase().replace(" ", "").replace(".", "").replace("_", "")
```

Comment:
Probably better to use a regexp identifier to replace all punctuation:

```javascript
return name.toLowerCase().replaceAll(/\p{P}/ug, "");
```

Reply:
done